src/app.js

```diff
@@ -29,24 +34,63 @@ app.use(
 
 app.use(express.static("public"));
 app.use(express.urlencoded({ extended: true }));
+app.use(setGa4ContainerId);
+app.use(setStatusCode);
+
+const routeInfo = [
+  {
+    path: "/",
+    pageTitle: "Home ",
+    taxonomyLevel1: "HomeTax1",
+    taxonomyLevel2: "HomeTax2",
+  },
+  {
+    path: "/service-description",
+    pageTitle: "Service Description ",
+    taxonomyLevel1: "ServiceTax1",
+    taxonomyLevel2: "ServiceTax2",
+  },
+  {
+    path: "/organisation-type",
+    pageTitle: "Organisation Type ",
+    taxonomyLevel1: "OrganisationTax1",
+    taxonomyLevel2: "OrganisationTax2",
+  },
+
+  {
+    path: "/choose-location",
+    pageTitle: "Choose Location",
+    taxonomyLevel1: "ChooseLocation1",
+    taxonomyLevel2: "ChooseLocation2",
+  },
+];
+routeInfo.forEach((route) => {
```

Comment:
I don't understand the goal of this loop. Can you explain me?

Reply:
Loop is so that I can set the taxonomy values for the different routes, otherwise wouldn't i have to call the app.use for each route individually ?

Reply:
Removed loop from app.js file , handled the path checking and setting of the values within the middleware function as recommended